pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -115,8 +116,9 @@ func (r *remoteImageService) listImagesV1(ctx context.Context, filter *runtimeap
 	resp, err := r.imageClient.ListImages(ctx, &runtimeapi.ListImagesRequest{
 		Filter: filter,
 	})
+	logger := klog.FromContext(ctx)
 	if err != nil {
-		klog.ErrorS(err, "ListImages with filter from image service failed", "filter", filter)
+		logger.Error(err, "ListImages with filter from image service failed", "filter", filter)
```

Comment:
`logger` is only used in error paths, better repeat it whenever needed instead paying the price at each call to `listImagesV1`